common/djangoapps/pipeline_js/templates/xmodule.js

```diff
@@ -4,21 +4,20 @@
 ## These files assume that several libraries are available and bound to
 ## variables in the global context, so we load those libraries with requirejs
 ## and attach them to the global context manually.
-define(["jquery", "underscore", "mathjax", "codemirror", "tinymce",
+define(["jquery", "underscore", "codemirror", "tinymce",
         "jquery.tinymce", "jquery.qtip", "jquery.scrollTo", "jquery.flot",
         "jquery.cookie",
         "utility"],
-       function($, _, MathJax, CodeMirror, tinymce) {
+       function($, _, CodeMirror, tinymce) {
     window.$ = $;
     window._ = _;
-    window.MathJax = MathJax;
+    require(['mathjax']);
```

Comment:
@andy-armstrong it would be difficult to write tests for the cases where we are using the mathjax as there are lot of places where we would need to test it. I came up with the only way that we need to run the tests like when mathajx is available and when it is not available but it would increase the time of tests on jenkins as we are doing the same thing on lms side as well. Can we just do manual testing for it ? cause it will make things more simple